Core/ALife.Core/Utility/EvoNumbers/EvoNumberExtensions.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
```

Comment:
Why not put these in the EvoNumber class file, or the IEvoNumber file?

Reply:
Interfaces can't have any method implementations in the version of .net we're on. This is a workaround to allow for some default implementations that any IEvoNumber will have available